stl/inc/istream

```diff
@@ -48,7 +48,7 @@ public:
     }
 
 protected:
-    __CLR_OR_THIS_CALL basic_istream(basic_istream&& _Right) : _Chcount(_Right._Chcount) {
+    __CLR_OR_THIS_CALL basic_istream(basic_istream&& _Right) noexcept(false) : _Chcount(_Right._Chcount) {

```

Comment:
We have a warning that suggests marking potentially-throwing functions `noexcept`??!? Wat? (No change requested.)

Reply:
Yeah, it is... not the greatest.